presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/suite/SuiteConfig.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.product.launcher.suite;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+
+public abstract class SuiteConfig
+{
+    private static final String DEFAULT_DOCKER_IMAGES_VERSION = "30";
+    private static final String DEFAULT_HADOOP_BASE_IMAGE = "prestodev/hdp2.6-hive";
+
+    /**
+     * export HADOOP_BASE_IMAGE="${HADOOP_BASE_IMAGE:-prestodev/hdp2.6-hive}"
+     */
+    public Optional<String> getHadoopBaseImage()
+    {
+        return Optional.of(DEFAULT_HADOOP_BASE_IMAGE);
+    }
+
+    public List<String> getExcludedGroups()
+    {
+        return ImmutableList.of();
+    }
+
+    /**
```

Comment:
drop the comments like this one eventually.

Reply:
I've kept it here for review to be simpler. I can drop them in a separate commit.

Reply:
Do you expect someone to actually read through all the definitions ? ;)